nextassist/src/main/java/fr/emse/pi/nextassist/model/Task.java

```diff
@@ -1,28 +1,41 @@
 package fr.emse.pi.nextassist.model;
 
 
+import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
 
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import java.time.Duration;
+import java.time.LocalDate;
 import java.time.LocalDateTime;
+import java.time.LocalTime;
 
 @Entity
 @Data
-@NoArgsConstructor
```

Comment:
Don't you need this one for TaskController?

Reply:
Yep, I've defined it within the task class. Take a look ;)